WordPress/Classes/ViewRelated/Tools/Time Zone/Views/TimeZoneTableViewCell.swift

```diff
@@ -0,0 +1,89 @@
+
+    let titleLabel = UILabel()
+
+    let leftSubtitle = UILabel()
+
+    let rightSubtitle = UILabel()
+
+    // MARK: - Initializers
+
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setupSubviews()
+    }
+
+    public required override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupSubviews()
+    }
+
+    private func setupSubviews() {
+        setupTimeZoneLabel(label: titleLabel, fontTextStyle: .body, textAlignment: .left, textColor: .label)
+        setupTimeZoneLabel(label: leftSubtitle, fontTextStyle: .caption1, textAlignment: .left, textColor: .secondaryLabel)
+        setupTimeZoneLabel(label: rightSubtitle, fontTextStyle: .caption1, textAlignment: .right, textColor: .secondaryLabel)
+
+        // Not every WPTimeZone has a time zone offset so wrapping content in UIStackView
+        // to allow for dynamic resizing for these cases
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+
+        let subtitleContainerView = UIView()
+        subtitleContainerView.translatesAutoresizingMaskIntoConstraints = false
+        subtitleContainerView.addSubview(leftSubtitle)
+        subtitleContainerView.addSubview(rightSubtitle)
+
+        stackView.addArrangedSubviews([titleLabel, subtitleContainerView])
+
+        NSLayoutConstraint.activate([
+            // contentView
+            contentView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            contentView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            contentView.topAnchor.constraint(equalTo: topAnchor),
+            contentView.bottomAnchor.constraint(equalTo: bottomAnchor),
+            // leftSubtitle
+            leftSubtitle.topAnchor.constraint(equalTo: subtitleContainerView.topAnchor, constant: Constants.subtitleTopPadding),
+            leftSubtitle.leadingAnchor.constraint(equalTo: subtitleContainerView.leadingAnchor),
+            leftSubtitle.bottomAnchor.constraint(equalTo: subtitleContainerView.bottomAnchor),
+            // rightSubtitle
+            rightSubtitle.centerYAnchor.constraint(equalTo: leftSubtitle.centerYAnchor),
+            rightSubtitle.leadingAnchor.constraint(greaterThanOrEqualTo: leftSubtitle.trailingAnchor, constant: Constants.subtitleHorizSpacingBetweenLabels),
+            rightSubtitle.trailingAnchor.constraint(equalTo: subtitleContainerView.trailingAnchor)
+        ])
+
+        contentView.addSubview(stackView)
+        contentView.pinSubviewToAllEdges(stackView, insets: UIEdgeInsets(
+                top: Constants.topPadding,
+                left: Constants.leadingTrailingPadding,
+                bottom: Constants.bottomPadding,
+                right: Constants.leadingTrailingPadding)
+        )
+    }
```

Comment:
This method is quite long: any chance we can break it up in different methods, for improved legibility?

Reply:
No problem; I've broken up the constraints layout into a new method